File: JavaAssignment-2/GreaterOf3.java
import java.util.*;
class GreaterOf3
{
public static void main(String[] args)
{
int num1,num2,num3;
System.out.println("Enter 3 numbers");
Scanner sc=new Scanner(System.in);
num1=sc.nextInt();
num2=sc.nextInt();
num3=sc.nextInt();
if((num1>num2)&&(num1>num3))
System.out.println(num1+" is greater");
else if((num1<num2)&&(num2>num3))
System.out.println(num2+" is greater");
else
System.out.println(num3+" is greater");
}
}

File: C++ Assignments/Assignment 2/greater of 3.cpp
#include<iostream>
using namespace std;
class Great
{
int num1,num2,num3;
public: void display() 
{
cout<<"Enter 3 numbers \n";
cin>>num1>>num2>>num3;
if((num1>num2)&&(num1>num3)) 
{
cout<<"Greater number is\n"<<num1;
}
else if((num2>num1)&&(num2>num3) ) 
{
cout<<"Greater number is\n"<<num2;
}
else
{
cout<<"Greater number is\n"<<num3;
}
}
};

int main()
{   
Great obj1;
obj1.display();
return 0;
}